src/Microsoft.Restier.EntityFramework/Model/ModelProducer.cs

```diff
@@ -4,7 +4,11 @@
 using System;
 using System.Collections.Generic;
 using System.Collections.ObjectModel;
+#if EF7
+using Microsoft.EntityFrameworkCore;
+#else
 using System.Data.Entity;
+#endif
```

Comment:
We add such code in EF project class, and remove EF7 related class, the question I have is can we do for all EF7 project class and make us does not EF7 project at all? Or it is a big effort?